release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,48 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	rm artifacts.zip 2>&1 >/dev/null
```

Comment:
I assume this shell redirection trick was meant to hide `rm` output. It does not hide stderr output because FD 1-associated file is not yet changed to /dev/null at the time of FD 2 redirection; FD 2 gets redirected to stdout _and stays there_ while FD 1 is then redirected to /dev/null.

```suggestion
	rm artifacts.zip > /dev/null 2>&1
```

Please fix all occurrences.